### PR TITLE
Upgrade to Storybook 6

### PR DESCRIPTION
**Pull request by MarcelGerber**
This upgrades our storybook version to 6.0, which is currently a release candidate but is scheduled to come out... tomorrow (https://github.com/storybookjs/storybook/issues/9311)!

This gives us out-of-the-box TS support and a nicer way to configure things.

I've also enabled some more addons using the new `@storybook/essentials` package:
* `addon-docs` auto-generates the props of our components
* `addon-actions` was enabled before
* `addon-controls` lets us control the props of the component to play around with it
* `addon-viewport` lets us view a component in different viewports

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "serve": "yarn tsn admin/server/server.tsx",
         "bake": "yarn tsn scripts/bakeSite.ts",
         "deploy-queue": "yarn tsn deploy/watch.ts",
-        "storybook": "yarn tsn node_modules/.bin/start-storybook -p 6006",
+        "storybook": "start-storybook -p 6006",
```
Comment (breck7) [inline package.json:34]: I think this means the new Storybook respects tsconfig settings. Neat!

File: stories/chartView.stories.tsx [hunk 2]
```diff
@@ -100,7 +100,8 @@ class ChartStoryView extends React.Component {
 }
 
 export default {
-    title: "ChartView"
+    title: "ChartView",
+    component: ChartView
```
Comment (breck7) [inline stories/chartView.stories.tsx:104]: What's this new component property?

Comment (MarcelGerber) [reply stories/chartView.stories.tsx:104]: This is used to auto-generate documentation for the props: https://github.com/storybookjs/storybook/blob/next/addons/docs/react/README.md#props-tables

Looks like this:
image

**Comment by MarcelGerber** [comment]
Just updated to the just-released storybook 6.0.0.
Merging this now.

**Pull request merged**